### ISSUE TITLE
tabindex greater than zero is not necessarily WCAG failure

### ISSUE THREAD

**Issue by jasonkiss** [dequelabs/axe-core#572]
Elements should not have tabindex greater than zero: the [explanation associated with this rule](https://dequeuniversity.com/rules/axe/2.4/tabindex) suggests it's a WCAG 2.0 A "MUST" requirement, and while Success Criterion 2.4.3 might be, not all positive tabindex values represent a failure of 2.4.3.

**Comment by WilcoFiers** [comment]
Yep, that's not correct. We'll get it sorted. Thanks for the message!
